packages/runtime/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@oazapfts/runtime",
+  "version": "0.0.0-development",
+  "description": "Runtime for OpenApi TypeScript client generator",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    },
+    "./query": {
+      "types": "./dist/query.d.ts",
+      "require": "./dist/query.cjs",
+      "import": "./dist/query.js"
+    },
+    "./util": {
+      "types": "./dist/util.d.ts",
+      "require": "./dist/util.cjs",
+      "import": "./dist/util.js"
+    },
+    "./headers": {
+      "types": "./dist/headers.d.ts",
+      "require": "./dist/headers.cjs",
+      "import": "./dist/headers.js"
+    }
+  },
+  "license": "MIT",
+  "repository": "oazapfts/oazapfts",
```

it's nice to include a `directory` here
```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/oazapfts/oazapfts",
    "directory": "packages/runtime"
  },
```